pkg/common/common.go

```diff
@@ -273,6 +273,9 @@ const (
 
 	// CDIControllerLeaderElectionHelperName is the name of the configmap that is used as a helper for controller leader election
 	CDIControllerLeaderElectionHelperName = "cdi-controller-leader-election-helper"
+
+	// ProgressSuffix is the suffix used for clone_progress metric
+	ProgressSuffix = "progress"
```

Comment:
@avlitman Is this a leftover?

Reply:
@sradco No, @akalenyu asked to make it a parameter. later it will also help us fix the prefix only in one place

Reply:
As discussed in private we will handle this change as part of https://github.com/kubevirt/containerized-data-importer/issues/2515.